arconia-dev/arconia-dev-services/arconia-dev-services-oracle-xe/src/main/java/io/arconia/dev/services/oracle/OracleXeDevServicesProperties.java

```diff
@@ -0,0 +1,88 @@
+package io.arconia.dev.services.oracle;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import io.arconia.dev.services.core.config.DevServicesProperties;
+
+/**
+ * Properties for the OracleXe Dev Services.
+ */
+@ConfigurationProperties(prefix = OracleXeDevServicesProperties.CONFIG_PREFIX)
+public class OracleXeDevServicesProperties implements DevServicesProperties {
+
+    public static final String CONFIG_PREFIX = "arconia.dev.services.oraclexe";
```

Comment:
I can see that in some places `oraclexe` is used in the property name, whereas `oracle-xe` in other places. Should we settle on `oracle-xe`?

Reply:
Good catch. Updated to "oracle-xe".